syntax = "proto3";
package bp;

message Bool {
    bool var = 1;
}

message Int32 {
    int32 var = 1;
}

message UInt32 {
    uint32 var = 1;
}

message Float {
    float var = 1;
}

message String {
    string var = 1;
}

message Bytes {
    bytes var = 1;
}

message Int32Ary {
    repeated int32 ary = 1;
}

message UInt32Ary {
    repeated uint32 ary = 1;
}

message FloatAry {
    repeated float ary = 1;
}
